docs/administrators/policy_cookbook.md

```diff
@@ -469,3 +469,106 @@ Regardless of the client, the server's log file will always report the error. No
   "server_type": "agent"
 }
 ```
+
+## Reusing policy to avoid duplication
+
+This example demonstrates how to reuse policy across multiple PEPs.
```

Comment:
Perhaps I should rephrase these?

Reply:
Perhaps ...
- Improving maintainability through Policy reuse
- Improving Policy through reusable rules
- Implementing maintainable Policy
- Implementing maintainable Policy through reusable rules

Thoughts?

Reply:
I think I'm leaning towards the fourth option. It captures the intent of the contents very well imo.

Reply:
Going with fourth option.